pkg/operator/configobserver/config_observer_controller.go

```diff
@@ -206,3 +202,13 @@ func (c *ConfigObserver) EventHandler() cache.ResourceEventHandler {
 		DeleteFunc: func(obj interface{}) { c.queue.Add(configObserverWorkKey) },
 	}
 }
+
+// updateStatus inspects the old and new status, sets the correct available and progressing conditions, and then issue the update if required.
+func (c *ConfigObserver) updateStatus(resourceVersion string, oldStatus, newStatus *operatorv1.OperatorStatus) (bool, error) {
+	if !reflect.DeepEqual(oldStatus, newStatus) {
```

Comment:
equalities

Reply:
equality

Reply:
semenatic equality or what do you mean here?